virttest/qemu_vm.py

```diff
@@ -284,13 +284,9 @@ def is_preconfig(self):
         """
         Return True if the qemu process is in preconfig status
         """
-        if self.is_dead():
-            return False
-        try:
-            self.verify_status("preconfig")
-            return True
-        except virt_vm.VMStatusError:
-            return False
+        if self.params.get_boolean("qemu_preconfig"):
```

Comment:
IMO, the value of the param and the vm status are two different thing, although the result is same sometimes.
I still suggest to checking vm status, and use different status name for different qemu version.

Reply:
It won't work for other cases. In all the cases, vm status is prelaunch at first. As a result, is_preconfig() returns true and jump to L3172, which is not what we expect.

Reply:
@huangyum If modified it this way, this whole function will be meaningless, why not calling `is_paused` directly instead of `is_preconfig` then?

Reply:
Because VM is always paused at this moment, and will return in L3186.

Reply:
@huangyum `if self.is_preconfig()` just now equals to `if self.params.get_boolean("qemu_preconfig") and self.is_paused()`, this function still seems meaningless to me. I mean, then this function should not be implemented this way.

Reply:
Do you mean to replace `if is_preconfig()` with` if params.get_boolean("qemu_preconfig") and is_paused()` ?

Reply:
agree with @vivianQizhu .

Reply:
> Do you mean to replace `if is_preconfig()` with` if params.get_boolean("qemu_preconfig") and is_paused()` ?

@huangyum That's not what I mean, I still not quite understand, the original issue seems to be the name of the status `preconfig` is modified in new version qemu, why we now just simply remove the checking? So previously the checking is totally meaningless from the beginning?

Reply:
With qemu<6.0, the status is preconfig at this point if preconfig is set, and prelaunch if not. So it's ok to check if is_preconfig by checking status. But with qemu6.0, the status is always prelaunch no matter it's preconfig or not. Seems checking the param is the only way to find out if it's preconfig.